static/sass/styles.scss

```diff
@@ -7,6 +7,9 @@
 // custom mixins
 @import "mixins/full-width-breakout";
 
+// import cookie policy
+@import "cookie-policy/build/css/cookie-policy";
```

Comment:
I don't think you need these. I believe the js brings in the CSS.

Reply:
I just double checked the JS, but without the css include it appeared unstyled.